data/data/bootstrap/vsphere/files/etc/NetworkManager/conf.d/99-ovirt.conf

```diff
@@ -0,0 +1,2 @@
+[main]
+rc-manager=unmanaged
```

Comment:
This file should be named `data/data/bootstrap/vsphere/files/etc/NetworkManager/conf.d/99-vsphere.conf`

Reply:
Oops! Fixed.